vigetlabs/wordpress-site-starter
# [#33] Build Tools update

# Summary

This PR does:
- Removes the old vite-serves from DDEV and is replaced with the new way of doing it. 
- Remove all the old vite-serve files
- Fixes Prettier - I thought this was running but not fully.
- Updates ddev node to 20
- Updates packages.json node to 20
- Adding Prettier to our custom ACF plugin
- Running Prettier on all the files

## Issues

* #33

## Testing Instructions

1. Stop ddev by running `ddev stop`.
2. Make sure you are running at least ddev version `v1.22.7` or newer. If not you can run `brew upgrade ddev/ddev/ddev`  if you install ddev with brew.
3. Start dev by running `bin/start`.  Make sure you don't have any errors. 
4. Checking that everything looks right. 

## Screenshots

NA
